erpnext/patches/v14_0/delete_education_module_portal_menu_items.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023, Frappe Technologies Pvt. Ltd. and Contributors
+# License: MIT. See LICENSE
+
+import frappe
+
+
+def execute():
+	doctypes = frappe.get_all("DocType", {"module": "education", "custom": 0}, pluck="name")
```

Comment:
@GursheenK it would be better to hardcode these DocTypes. Many existing sites won't have the education module installed. In this case your DocTypes variable will be an empty list and the obsolete portal menu items will remain there.